contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -148,42 +149,54 @@ contract WithdrawalQueue {
     ShareRate[] public finalizationRates;
 
     /**
-     * @param _owner address that will be able to invoke `restake` and `finalize` methods.
      * @param _stETH address of StETH contract
      * @param _wstETH address of WstETH contract
      */
-    constructor(address payable _owner, address _stETH, address _wstETH) {
-        if (_owner == address(0)) revert ZeroOwner();
-
+    constructor(address _stETH, address _wstETH) {
         // init immutables
         STETH = _stETH;
         WSTETH = _wstETH;
-        OWNER = _owner;
 
-        // petrify the implementation by assigning a zero Lido agent address
-        _initialize(address(0));
+        // petrify the implementation by assigning a zero addresses for every role
+        _initialize(address(0), address(0), address(0), address(0));
     }
 
-    function initialize(address _lidoDAOAgent) external {
-        if (_lidoDAOAgent == address(0)) {
-            revert LidoDAOAgentZeroAddress();
+    /**
+     * @param _admin admin address that can change every role.
+     * @param _pauser address that will be able to pause the withdrawals
+     * @param _resumer address that will be able to resume the withdrawals after pause
+     * @param _finalizer address that can finalize requests in the queue
+     * @dev Reverts with `AdminZeroAddress()` if `_admin` equals to `address(0)`
+     */
+    function initialize(address _admin, address _pauser, address _resumer, address _finalizer) external {
```

Comment:
Should the contract be in "paused" state upon deployment and initialization? If so, let's mention it explicitly.